Source/Configuration/YPImagePickerConfiguration.swift

```diff
@@ -272,6 +272,8 @@ public struct YPConfigLibrary {
 
     /// Allow selection of both photo and video from the photo gallery.
     public var allowPhotoAndVideoSelection : Bool = false
+
+    public var assetPreviewFooterView: UIView?
```

Comment:
nit: Can we add some documentation to this just to be consistent with other properties in the config?

Reply:
Oh yes, I forgot to do that. Updated now.